public/app/plugins/datasource/loki/sortDataFrame.ts

```diff
@@ -85,10 +85,12 @@ export function sortDataFrameByTime(frame: DataFrame, dir: SortDirection): DataF
     ...rest,
     fields: fields.map((field) => ({
       ...field,
-      values: new SortedVector(field.values, index).toArray(),
-      nanos: field.nanos === undefined ? undefined : new SortedVector(field.nanos, index).toArray(),
+      values: sorted(field.values, index),
```

Comment:
@gabor -- can you verify that this still does what you expect?  It is the only use of SortedVector

Reply:
I did. LGTM!